lib/backend/sanitize.go

```diff
@@ -36,11 +37,25 @@ const errorMessage = "special characters are not allowed in resource names, plea
 var allowPattern = regexp.MustCompile(`^[0-9A-Za-z@_:.\-/+]*$`)
 
 // denyPattern matches some unallowed combinations
-var denyPattern = regexp.MustCompile(`//`)
+var denyPatterns = []string{
```

Comment:
The fact that `allowPattern` is a regexp and `denyPatterns` is just substring match is a bit confusing, no?

Reply:
This one is partly on me, I asked for the substring matching as it seemed simpler to reason about than a regex with various `|` components.